jd-geek-source/src/app/app.component.scss

```diff
@@ -0,0 +1,92 @@
+@import '../constants/gutters.scss';
+.main-gutter {
+  column-gap: $MAIN_GUTTER_WIDTH;
+}
+
+.half-gutter {
+  column-gap: $MAIN_GUTTER_WIDTH/2;
+}
+
+.double-gutter {
+  column-gap: $MAIN_GUTTER_WIDTH*2;
+}
+
+.toolbar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  background-color: #123e69;
+  color: white;
+  font-weight: 600;
+}
+
+.toolbar img {
+  margin: 0 16px;
```

Comment:
`16px` and `0 16px` appear multiple times. I wonder if this is an opportunity to leverage a variable to keep these units in sync? In other words, what happens if you that value of `16px` needs to be changed to `20px` in the future? Will I have to update all five uses of `16px`? If so, I should extract that to a variable so I only have to update the variable for all the associated elements to get the proper styling.